### PR TITLE
update json schema

### PR DESCRIPTION
**Pull request by Mearman**
for version regex see: https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string

### CODE REVIEW

**Comment by Mearman** [comment]
hmmm, should probably have a an associated changeset

File: packages/breadboard/breadboard.schema.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "https://raw.githubusercontent.com/breadboard-ai/breadboard/main/packages/breadboard/breadboard.schema.json",
+  "$id": "https://raw.githubusercontent.com/breadboard-ai/breadboard/@google-labs/breadboard@0.8.0/packages/breadboard/breadboard.schema.json",
```
Comment (aomarks) [inline packages/breadboard/breadboard.schema.json:3]: I'm a bit skeptical we'll keep this up to date with the npm version unless we automate it. But also, this is unlikely to change as frequently as our version numbers.

So maybe we should have independent versioning for the json schema?

(I guess also, what is the practical implication of $id? what actually matters is what URL people put in their $schema, and to control that maybe we should use git tags?)

File: packages/breadboard/breadboard.schema.json [hunk 2]
```diff
@@ -10,16 +10,42 @@
...
+      "type": "string"
+    },
+    "description": {
+      "description": "",
+      "type": "string"
+    },
+    "version": {
+      "description": "A version string for the graph",
+      "type": "string",
+      "pattern": "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"
```
Comment (aomarks) [inline packages/breadboard/breadboard.schema.json:37]: This is semver, I assume? Maybe just mention that in the description

**Review by aomarks — COMMENTED** [review]
Cool!

File: packages/breadboard/breadboard.schema.json (hunk 2 above)
Comment (Mearman) [reply packages/breadboard/breadboard.schema.json:37]: I even thought about updating it, then got distracted by the changesets

**Comment by Mearman** [comment]
stale now

**Pull request closed without merging**